src/app/profile/overview/overview.component.html

```diff
@@ -27,7 +27,7 @@ <h3 class="col-sm-12 profile-wrapper-username">{{context?.user?.attributes?.full
           </p>
         </ng-template>
         <ng-template #addEmail>
-          <p class="col-sm-12 profile-wrapper-email">
+          <p class="col-sm-12 profile-wrapper-email" *ngIf="!context.user.attributes.emailPrivate">
```

Comment:
This is problematic because there are no elvis operators on the properties - try logging out on this page and watch the console to see what I mean.  The context user pushes an empty object so this will cause failures,

Reply:
@dgutride I could reproduce any console errors on my local machine or the cd snapshot. Can you please explain what problems or errors could happen because of this change?

Reply:
@dgutride Since context is always set with user data in overview.component.ts even if viewing a user page without login, do we need elvis operators on properties? On the same page i can see a few places that we've used elvis operators and on the others not. Is there a reason for that? Do let me know if I am missing something or some corner cases where context is not defined which will throw console errors.

Reply:
@rohitkrai03  - looks like you are correct in this instance, the context.user is now using the new property I added last month called currentLoggedInUser.  If it was pointing at loggedInUser from the subscription above, you'd need to protect that value or it would fail.  I'll approve this one.